libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_syncing_packet_handler.cpp

```diff
@@ -141,12 +141,10 @@ void ExtSyncingPacketHandler::requestPendingDagBlocks(std::shared_ptr<TaraxaPeer
 
 void ExtSyncingPacketHandler::requestDagBlocks(const dev::p2p::NodeID &_nodeID,
                                                const std::unordered_set<blk_hash_t> &blocks, uint64_t period) {
-  LOG(log_nf_) << "Sending GetDagSyncPacket";
-  dev::RLPStream s(blocks.size() + 1);  // Period + block itself
-  s << static_cast<uint64_t>(period);   // Send period first
-  for (const auto &blk : blocks) {
-    s << blk;
-  }
+  dev::RLPStream s(2);  // Period + blocks list
```

Comment:
I did create issue for this as it would be nice to fix it for every packet handler and the solution might be more complicated than just creating constant for each packet.... You can read more here https://github.com/Taraxa-project/taraxa-node/issues/1830